mlflow/sagemaker/__init__.py

```diff
@@ -464,6 +469,7 @@ def _deploy(role, image_url, app_name, model_s3_path, run_id, region_name, mode,
                                           run_id=run_id,
                                           instance_type=instance_type,
                                           instance_count=instance_count,
+                                          vpc_config=vpc_config,
```

Comment:
Maybe add the new parameter at the end so as not to break any existing code that depends on parameter order.

Reply:
Since `_update_sagemaker_endpoint` is an internal method that existing code outside of this module should not depend on, it seems safe to insert the parameter in its current location.  This way, the method's parameters are structured such that all of the arguments that will be passed to the boto3 clients are grouped together.